e2e/page-objects/onboarding.ts

```diff
@@ -5,11 +5,7 @@ export default class Onboarding {
 	constructor( private page: Page ) {}
 
 	private get locator() {
-		// This fails, not sure why...
-		// return this.page.getByTestId( 'onboarding' );
```

Comment:
My guess is that the unexpected failure may have related to attempting queries before the UI was visible. With the new approach proposed in these changes, the original query appears to work consistently.